### DISCUSSION TITLE
Naive question - can Faikin Auto mode be controlled by HA?

### DISCUSSION THREAD

**Discussion by PianSom** [revk/xESP32-Faikout#308] [General]
Title says it all. I like Faikin Auto mode, but would prefer it to be enabled/disabled/times set by Home Assistant. Is this a possibility?

**Comment by revk** [comment]
Well you can do it all externally sending temp range and current temp using `control` mqtt message - not idea if HA can do that.

**Reply by PianSom** [reply]
It certainly could do it, if set up correctly. But I was - lazily - hoping to piggy-back off the existing excellent code :)
